imports/components/@primitives/UI/live/index.js

```diff
@@ -113,11 +131,28 @@ const map = ({ live }) => ({ live });
 
 const withRedux = connect(map);
 
+// const withLive = graphql(LIVE_QUERY, {
+//   options: { pollInterval: 60000 },
+// });
+
+// XXX FOR TESTING ONLY
 const withLive = graphql(LIVE_QUERY, {
```

Comment:
We'll need to comment this out for beta. Just leave it in the file to make it easy to change for local testing. This'll have to be QA'ed on a service day